app/src/main/java/org/walleth/data/AppDatabase.java

```diff
@@ -12,7 +12,7 @@
 import org.walleth.data.transactions.TransactionDAO;
 import org.walleth.data.transactions.TransactionEntity;
 
-@Database(entities = {AddressBookEntry.class, Token.class, Balance.class, TransactionEntity.class}, version = 1)
+@Database(entities = {AddressBookEntry.class, Token.class, Balance.class, TransactionEntity.class}, version = 2)
```

Comment:
why do we go to v2 here?

Reply:
Oh, sorry. Thanks for spotting. I was to lazy to restore all my accounts on my device after doing an upgrade on a different branch. (#2 to the rescue :-)